### ISSUE TITLE
Sledování zásilek

### ISSUE THREAD

**Issue by petrcernansky** [pavelevap/ceske-sluzby#102]
Zdravím, bylo by fajn u sledování zásilke přidat 2 věci:
1. aby se číslo zásilky zobrazilo v uživatelském profilu.
2. možnost hromadného importu čísel pro sledování zásilky - u kurýrních služeb (u PPL a GLS odzkoušeno) není problém vygenerovat např. csv soubor, kde v jednom sloupci bude číslo objednávky (to z woocommerce) a ve druhém číslo pro sledování zásilky. 

Bylo by to možné? Při např. 40ti objednávkách není reálné to vkládat ručně ke každé zásilce.

**Comment by pavelevap** [comment]
ad 1) Ano, to by tam určitě mělo být, mám to ještě v poznámkách zde: https://github.com/pavelevap/ceske-sluzby/issues/14#issuecomment-154694810

ad 2) Zajímavý nápad, ale má to vůbec cenu? Myslím to tak, jestli už samotní dopravci nějak nezasílají informace o odeslané zásilce (SMS a potom přímo volají)? A při importu by to mělo zrovna automaticky odeslat notifikační email? Ještě jsem to moc nezkoumal, ale nemají nějaké API, kde by to šlo automaticky synchronizovat? PPL Parcelshop má myslím jen nějaké API pro získání a zobrazení poboček, ale nevím zatím přesně... Jinak by asi možnost hromadného importu šla připravit, ideálně se jen napojit na nějaký existující importní plugin pro WooCommerce pomocí nějaké jednoduché funkce...

**Comment by petrcernansky** [comment]
ad 2) Ano kurýři posílají sms a volají. Nicméně občas někdo neuvede tel. číslo, nebo jej uvede špatně. Dále se stává (a to bohužel často) že nějaká zásilka uvízne někde na depu a nebo že se kurýr nemohl klientovi dovolat a někde to leží. A bylo by fajn kdyby si klient sám mohl skontrolovat kde zásilka právě je. 

Testuji nyní WP All Import, ale to celkém složitý plugin a ještě jsem bez výsledku.

**Comment by novetrendy** [comment]
WP All Import/Export je nejlepší a podle mě i nejjednodušší plugin pro import/export. Zatím jsem nenarazil na nic, co by nešlo importovat. Pokud člověk neví do jakého custom field má co importovat je nejjednodušší si vyexportovat třeba 10 položek a hned ví, co které pole znamená. Nejlepší je ovšem cron, několik eshopů aktualizuji několikrát denně (ceny, sklad) a to jak z účetnictví tak z feedů od dodavatelů.

**Comment by pavelevap** [comment]
@novetrendy: Souhlas, WP All Export jsem chtěl původně použít pro XML feedy, ale nakonec jsem tam nenašel potřebné hooky a nastavení bylo pro běžného uživatele dost složité. Ale ještě to časem prověřím, bylo by to ideální řešení :-)

**Comment by petrcernansky** [comment]
No, tak se zeptám, nyní mi stačí ano/ne, ať vím jestli to vůbec zkoušet - jde přes WP All Import naimportovat data do custom field k určité objednávce? A @pavelevap - jak je pojmenováno custom field ke sledování objednávky? 

Další věc je ještě volba dopravce z rozevírací nabídky...

**Comment by pavelevap** [comment]
@petrcernansky: Snad to nebude problém, ale neukládám to jako custom field, protože mi to přišlo vhodnější a logičtější jako term field k použitému shipping item (a nikoli k celé objednávce).

https://github.com/pavelevap/ceske-sluzby/blob/80a0256a5414ddf873483ee6616ac0f4891b6cd7/includes/class-ceske-sluzby-sledovani-zasilek.php#L140

Ale koukám, že bych tím mohl asi narušit různá workflow :-( Každopádně by šla asi pro WP All Import napsat funkce, která by to také snadno uměla, ale musím se na to nejdříve podívat...
